database/migrations/2012_10_03_133633_create_base_tables.php

```diff
@@ -286,8 +286,8 @@ public function up(): void
                     $table->bigIncrements('id')->first();
                 }
 
-                $table->string('User', 32)->index();
-                $table->string('Friend', 32)->index();
+                $table->string('User', 32)->index('user_relations_user_id_index');
+                $table->string('Friend', 32)->index('user_relations_related_user_id_index');
```

Comment:
Same comment. There are `user_id` and `related_user_id` columns in the prod db.

Reply:
this was a mistake - those should be username instead of user_id if anything. both User and Friend columns will be removed in the future, so adding an index name for those isn't necessary either.
removed custom index names in https://github.com/RetroAchievements/RAWeb/pull/1935/commits/b78ec3a435a8a5474e20a149ae209372d2b14b89